install_config/aggregate_logging.adoc

```diff
@@ -782,6 +830,56 @@ Please note that changes to the configmap might not appear until after redeployi
 the pods. Persisting these changes across deployments can be accomplished by setting
 `openshift_logging_es_allows_cluster_reader` to `true` in the inventory file.
 
+[[expose-elasticsearch-as-route]]
+*Expose Elasticsearch as a Route*
+
+By default, Elasticsearch deployed with OpenShift aggregated logging is not
+accessible from outside the logging cluster. You can enable a route for external
+access to Elasticsearch for those tools that want to access its data.
+
+You now have direct access to Elasticsearch using only your OpenShift token and
+have the ability to provide the external Elasticsearch and Elasticsearch Ops
+hostnames when creating the server certificate (similar to Kibana). Ansible
+tasks now simplify route deployment.
```

Comment:
The above sounds like a results para. Is this out of place? Or is it describing the actions that are currently available? If the latter, then it might need to be more clear.